typescript.js

```diff
@@ -9,15 +9,19 @@ module.exports = {
       parserOptions: { warnOnUnsupportedTypeScriptVersion: false },
       extends: ['plugin:@typescript-eslint/recommended'],
       rules: {
-        'no-use-before-define': 'off', // Turning off because we have the TS version off
-        '@typescript-eslint/no-use-before-define': 'off',
-        '@typescript-eslint/explicit-function-return-type': ['error', { allowExpressions: true }],
-
+        'no-use-before-define': 'off', // @typescript-eslint/no-use-before-defined requires this to be off
+        '@typescript-eslint/no-use-before-define': ['error', { functions: false }],
         'no-unused-vars': 'off', // @typescript-eslint/no-unused-vars requires this to be off
-        '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Comment:
The default for this is "@typescript-eslint/no-unused-vars": ["warn"]. We don't need the argsIgnorePattern, and we don't have that for normal js files.

You can still add an underscore for beginning params that you don't use in a function like for a req, res function. That isn't linted by this rule.
![image](https://user-images.githubusercontent.com/77301861/215881581-ce14c6c1-17a4-46e9-9ef8-daf2f689106b.png)
![image](https://user-images.githubusercontent.com/77301861/215881591-7b8b0670-6287-40f7-92ed-2576f1a22493.png)